content/docs/practices/histograms.md

```diff
@@ -0,0 +1,220 @@
+duration has its sharp spike at 320ms and almost all observations will
+fall into the bucket from 300ms to 450ms. The 95th percentile is
+calculated to be 442.5ms, although the correct values is close to
+320ms. While you are only a tiny bit outside of your SLA, the
+calculated 95th quantile looks much worse.
+
+A summary would have had no problem calculating the correct percentile
+value in both cases, at least if it uses an appropriate algorithm on
+the client side (like the [one used by the Go
+client](http://www.cs.rutgers.edu/~muthu/bquant.pdf)). Unfortunately,
+you cannot use a summary if you need to aggregate the observations
+from a number of instances.
+
+Luckily, due to your appropriate choice of bucket boundaries, even in
+this contrived example of very sharp spikes in the distribution of
+observed values, the histogram was able to identify correctly if you
+were within or outside of your SLA. Also, the closer the actual value
+of the quantile is to our SLA (or in other words, the value we are
+actually most interested in), the more accurate the calculated value
+becomes.
+
+Let us now modify the experiment once more. In the new setup, the
+distributions of request durations has a spike at 150ms, but it is not
+quite as sharp as before and only comprises 90% of the
+observations. 10% of the observations are evenly spread out in a long
+tail between 150ms and 450ms. With that distribution, the 95th
+percentile happens to be exactly at our SLA of 300ms. With the
+histogram, the calculated value is accurate, as the value of the 95th
+percentile happens to coincide with one of the bucket boundaries. Even
+slightly different values would still be accurate as the (contrived)
+even distribution within the relevant buckets is exactly what the
+linear interpolation within a bucket assumes.
+
+The error of the quantile reported by a summary gets more interesting
+now. The error of the quantile in a summary is configured in the
+dimension of φ. In our case we might have configured 0.95±0.01,
+i.e. the calculated value will be between the 94th and 96th
+percentile. The 94th quantile with the distribution described above is
+270ms, the 96th quantile is 330ms. The calculated value of the 95th
+percentile reported by the summary can be anywhere in the interval
+between 270ms and 330ms, which unfortunately is all the difference
+between clearly within the SLA vs. clearly outside the SLA.
+
+The bottom line is: If you use a summary, you control the error in the
+dimension of φ. If you use a histogram, you control the error in the
+dimension of the observed value (via choosing the appropriate bucket
+layout). With a broad distribution, small changes in φ result in
+large deviations in the observed value. With a sharp distribution, a
+small interval of observed values covers a large interval of φ.
+
+Two rules of thumb:
+
+  1. If you need to aggregate, choose histograms.
+
+  2. Otherwise, choose a histogram if you need accuracy in the
```

Comment:
This is a bit wordy for a rule of thumb.

Maybe something along the lines of:
choose a histogram if you have an idea of the range/distribution of values that will be seen and have a good idea of the buckets to use. Choose a summary if you need an accurate quartile, no matter what the range/distribution of the values is.